src/quickmozview.cpp

```diff
@@ -128,7 +128,7 @@ void QuickMozView::createGeckoGLContext()
 
 void QuickMozView::requestGLContext(bool& hasContext, QSize& viewPortSize)
 {
-    hasContext = false;
+    hasContext = true;
```

Comment:
Previously EmbedLitePuppetWidget was always returning true. This matches to the change in https://github.com/nemomobile-packages/gecko-dev/pull/5/files#diff-505e1dcec320a20cbb6bae62dc4b5ca5L569

Strictly speaking this change is not needed as nsIWindow::HasGLContext is only called from CompositorOGL.cpp in code that is behind "MOZ_USE_EXTERNAL_WINDOW" environment variable (in embedlite 31).